skimage/measure/fit.py

```diff
@@ -428,6 +428,10 @@ def estimate(self, data):
         float_type = np.promote_types(data.dtype, np.float32)
         data = data.astype(float_type, copy=False)
 
+        # shift data
+        origin = data.mean(axis=0)
+        data = data - origin
+
```

Comment:
If we manipulate the input data anyway, why should we stop at only shifting the origin? We could normalize the amplitude as well. That way we'd even achieve a robust fit for very small values in `data`?

It would also be useful if the comment could explain why the manipulation is necessary. E.g. "normalize value range to avoid misfitting due to numeric errors if differences between data points a very small".

Reply:
Now, it also applies the normalization.